crates/fluvio-smartengine/src/engine.rs

```diff
@@ -24,7 +24,9 @@ pub struct SmartEngine(Engine);
 #[allow(clippy::new_without_default)]
 impl SmartEngine {
     pub fn new() -> Self {
-        Self(Engine::default())
+        let mut config = wasmtime::Config::default();
+        config.consume_fuel(true);
+        Self(Engine::new(&config).expect("Config is static"))
```

Comment:
should remove expect

Reply:
This expect can be tested before deploy.

Reply:
As the config is static.

Reply:
So success of Engine::new().expect() is unit tested.

Reply:
the default() that was being called previously had an unwrap() inside anyway
```
impl Default for Engine {
    fn default() -> Engine {
        Engine::new(&Config::default()).unwrap()
    }
}
```

Reply:
Yes but it can't fail.. it's a static configuration

Reply:
Which is unit tested

Reply:
Fair enough